docs/step-by-step-guide.md

```diff
@@ -48,6 +48,33 @@ Cleaner input data ensures that the model works with reliable and consistent inf
 
    - Ensures the document is clean and ready for the next step.
 
+### 🔍 **API Example**
```

Comment:
@stefanfrench not sure if this is the best format but I put a quick example to test how it would look